src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,22 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = [];
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState,
+  reducers: {
+    checkCategories(state) {
+      let newState = '';
+      if (state === []) {
+        newState = 'Under construction';
+      } else {
+        newState = state;
+      }
+      return newState;
+    },
```

Comment:
- [ ] Kindly note that the chechCategories should also have a payload and that payload should be what we use to check if it is in accordance with what we expect.
It can look like: 
```
checkStatus: (state, action) => {
      state.categories =
        action.payload === "Under construction"
          ? "Under construction"
          : state.categories
    },
```